demo/src/Alice.ts

```diff
@@ -8,7 +8,7 @@ export class Alice extends BaseAgent {
   public connectionRecordFaberId?: string
 
   public constructor(port: number, name: string) {
-    super(port, name)
+    super({ port, name, useSharedComponents: false })
```

Comment:
I just would like to settle also in GitHub my feelings about using this term because I know it's clear for everyone involved in this for some time that it came from 'Indy shared components'. But I don't want to commit the same mistake of carrying the legacy forever and confuse newcomers (I'm basing these words on my own experience).

For the time being I think it's nice to reflect it in this demo and be explicit that we support both the Indy SDK and the 'replacement shared components', as this is a major stepping stone for AFJ and we are super excited about it.

But in a near future I would like to simply set a demo based in Askar or the InMemoryWallet, showing the exchange of AnonCreds and JSON-LD credentials and making this modular architecture to seem natural and clearly ledger/credential format agnostic.

Reply:
Good point, fully agree!

Reply:
I've changed the parameter name to `useLegacyIndySdk` for now